### PR TITLE
chore(ci): improve npm release procedure

### PR DESCRIPTION
**Pull request by soonum**

### CODE REVIEW

File: .github/workflows/make_release.yml [hunk 1]
```diff
@@ -34,6 +39,10 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Create NPM version tag
+        run: |
+          echo "NPM_TAG=$(git describe --tags --abbrev=0 | tr -d 'tfhe-rs-')" >> "${GITHUB_ENV}"
+
```
[the remark this answers is not in the document]
Comment (soonum) [reply .github/workflows/make_release.yml:45]: You're right.

**Comment by soonum** [comment]
We need to rebase, after merging of pull_request_target removal, to have a green CI.

**Comment by IceTDrinker** [comment]
CI looks to be starting, we'll see

File: .github/workflows/make_release.yml [hunk 2]
```diff
@@ -34,6 +39,10 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Create NPM version tag
+        run: |
+          echo "NPM_TAG=$(sed -n -e '1,/^version/p' tfhe/Cargo.toml | grep '^version[[:space:]]*=' | tr -d 'version =')" >> "${GITHUB_ENV}"
```
Comment (IceTDrinker) [inline .github/workflows/make_release.yml:44]: 'version =' does not match the grep regex, do you want to go ahead like this ?

Comment (soonum) [reply .github/workflows/make_release.yml:44]: What do you mean it doesn't match ? The missing repetition of `[[:space:]]` ?

Comment (IceTDrinker) [reply .github/workflows/make_release.yml:44]: that's why we use cut in the Makefile

**Review by IceTDrinker — APPROVED** [review]
Thanks !

**Pull request merged**